packages/core/src/indexing/migrations/1-create-model-table.ts

```diff
@@ -100,6 +142,5 @@ export async function createConfigTable(dataSource: Knex, tableName: string, net
       break
     default:
       throw new Error(`Invalid config table creation requested: ${tableName}`)
-      process.exit(-1)
```

Comment:
Why not exit here; seems like this would be a good exit point during node startup?

Reply:
It's unreachable code, nothing gets executed after a `throw`